wsireg/reg_images/np_reg_image.py

```diff
@@ -70,27 +88,28 @@ def read_reg_image(self):
 
         self.image = np.squeeze(self.image)
 
-        image = sitk.GetImageFromArray(self.image)
+        self.image = sitk.GetImageFromArray(self.image)
```

Comment:
I notice that this bit of code will override `self.image` from a numpy array to a `sitk.Image`. I'm not sure if this class is exposed to end users, but it could lead to some hard to track bugs if the type of an attribute depends on what methods have been called. Just something to think about.

Reply:
This is a great point . I think I will simply return the preprocessed image here rather than replacing the original array. My concern for this was memory. If you had passed a very large array, then the reg image will be a slightly smaller array but could eat up memory rather quickly. I think moving swiftly on #5 is where I'd like to start before refactoring all of this.

Reply:
Makes sense, I had some work on #5 yesterday and I'll try to push that forward by the end of the week!